admin-ui/app/utils/ApiKeyRedirect.js

```diff
@@ -42,33 +44,48 @@ function ApiKeyRedirect({
                     height: 'auto',
                   }}
                   src={require('Images/gif/loader.gif')}
-                  alt="loading..."
+                  alt='loading...'
                 />
-                <div className="initial-loader__row">
-                  Redirecting...
-                </div>
+                <div className='initial-loader__row'>Redirecting...</div>
               </div>
             )}
-        
-        {!backendIsUp && 
-        <GluuErrorModal
-          message={'The UI backend service is down'}
-          description={'It may due to any of the following reason <br/>1. Admin UI Backend is down. <br/>2. Unable to get license credentials from Gluu server.<br/>Please contact the site administrator or check server logs.'}
-        />
-        }
+          </>
+        )}
+
+        {!backendIsUp && (
+          <GluuErrorModal
+            message={'The UI backend service is down'}
+            description={
+              'It may due to any of the following reason <br/>1. Admin UI Backend is down. <br/>2. Unable to get license credentials from Gluu server.<br/>Please contact the site administrator or check server logs.'
+            }
+          />
+        )}
 
-        {roleNotFound && 
-        <GluuErrorModal
-          message={t('Unauthorized User')}
-          description={'The logged-in user do not have valid role. Logging out of Admin UI'}
-        />
-        }
+        {isTimeout && (
+          <GluuErrorModal
+            message={'The UI backend service is down'}
+            description={
+              'The request has been terminated as there is not response from server for more than 60 seconds.'
```

Comment:
The request has been terminated as there is no response from the server for more than 60 seconds.